### PR TITLE
adds the first api

### PR DESCRIPTION
**Pull request by Vondry66**
Unsure if this is the correct way of testing for it.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const express = require("express");
+const { getCategories } = require("./controllers/categories.controller");
+const app = express();
+
+app.use(express.json());
```
Comment (Rayhaan2k9) [inline app.js:5]: We don't need this just yet!

**Review by Rayhaan2k9 — CHANGES_REQUESTED** [review]
I can't see a test file. Please can you write tests for your first endpoint and then send another Pull request? :)

File: app.js (hunk 1 above)
Comment (Vondry66) [reply app.js:5]: Sorry forgot to push the tests

**Review by Augs0 — APPROVED** [review]
Lovely job on the tests. Remember, as Rayhaan said, there is no need to include things like `app.use(express.json());` This is jumping ahead to future tickets. Please remove this until you reach the POST ticket.

**Pull request merged**